psycopg3/connection.py

```diff
@@ -110,6 +122,10 @@ def autocommit(self, value: bool) -> None:
     def _set_autocommit(self, value: bool) -> None:
         # Base implementation, not thread safe
         # subclasses must call it holding a lock
+        if len(self._savepoints) > 0:
+            raise e.ProgrammingError(
+                "can't change autocommit state when in Transaction context"
```

Comment:
```suggestion
                "couldn't change autocommit state: in Transaction context"
```
To match new wording below.
Not sure what the best way to refer to "Transaction context" is. Maybe "connection.transaction() context" would more concretely relate to the API invoked by user?

Reply:
Seems good :+1: Postgres has a clear distinction between "can not" (something impossible to obtain) and "could not" (something that it that moment wasn't possible to do) and I'd like to follow the same guideline.